erpnext/accounts/doctype/chart_of_accounts_importer/chart_of_accounts_importer.py

```diff
@@ -352,15 +352,15 @@ def get_mandatory_account_types():
 
 def validate_account_types(accounts):
 	account_types_for_ledger = ["Cost of Goods Sold", "Depreciation", "Fixed Asset", "Payable", "Receivable", "Stock Adjustment"]
-	account_types = [accounts[d]["account_type"] for d in accounts if not accounts[d]['is_group'] == 1]
+	account_types = [accounts[d]["account_type"] for d in accounts if not accounts[d]['is_group'] == '1']
```

Comment:
cast user input to integer using `cint` or `int` instead? We don't use string for checkbox fields anywhere else.

Reply:
Yeah have to discuss more on this issue with Deepesh, can this PR be put on hold?

Reply:
You can make it "Draft" 😄